README.md

```diff
@@ -1,9 +1,40 @@
-[![Build Status](https://travis-ci.org/CityOfZion/cityofzion-website.svg?branch=master)](https://travis-ci.org/CityOfZion/cityofzion-website)
-
-The cityofzion.io website source: https://cityofzion.io
-
-Very simple, no JS necessary. Using [Hugo](https://gohugo.io/), a fast static website generator.
-
+<p align="center">
+  <img 
+    src="http://res.cloudinary.com/vidsy/image/upload/v1503160820/CoZ_Icon_DARKBLUE_200x178px_oq0gxm.png" 
+    width="125px"
+    alt="City of Zion logo">
+</p>
+
+<h1 align="center">CityOfZion.io source</h1>
+
+<p align="center">
+  The CityOfZion.io website source: https://cityofzion.io
+</p>
+
+<p align="center">
+	<a href="https://travis-ci.org/CityOfZion/cityofzion-website">
+		<img src="https://travis-ci.org/CityOfZion/cityofzion-website.svg?branch=master"/>
+	</a>
+</p>
+
+## Overview
+Very simple, no JS necessary. Using [Hugo](https://gohugo.io/), a fast static website generator and [NodeJS](https://nodejs.org) for building.
+
+## Adding yourself to the team
```

Comment:
i don't think this should be at the top of the file. perhaps better after the run instructions

Reply:
nah i meant that how-to add yourself the to team instructions should go below the "how to run the dev server"

Reply:
It's changed now